addon/content/message.jsx

```diff
@@ -7,19 +7,20 @@
            MessageDetails, MessageNotification */
 /* exported Message */
 
+const { isAccel } = ChromeUtils.import(
```

Comment:
At some stage we're not going to be able to import scripts from the back-end into message content, so I think it'd be nice if we start working out how we're going to handle these kind of simple functions. 

Maybe we should just duplicate it for now, as far as I know there's nothing in general html/javascript land that will help us here, unless there's something React has?

Reply:
There's `window.navigator.platform`. We could just unconditionally listen for ctrl or cmd (I don't use a Mac, so I don't know if this would cause surprises or not...).

In the long term, the Redux store should store the OS and the `isAccel` function should be imported as an es import. I can't figure out how to do es imports in the `stub.html` code, though.

Reply:
Ah yes we could use `navigator.platform`. I think we could just duplicate isAccel here, it isn't really worth a utility function on its own IMO.

At the moment we probably can't do ES imports, I suspect we're missing bits from the Thunderbird core to be able to do that. Though it might be possible to use if the original javascript file has type="module" in the script tag in the html file, but we probably can't move everything across yet, because of all the ChromeUtils.import calls.

Reply:
We can't do imports in `xhtml`, it seems...it has to be regular `html`...